components/pedigree/resources/src/main/resources/pedigree/view/cancersLegend.js

```diff
@@ -6,52 +6,75 @@
+                                         "HP:0100751": "#BC8F8F",  // (?) rosybrown
+                                         "HP:0100031": "#FFFF00",  // yellow
+                                         "HP:0002896": "#770000",  // dark red
+                                         "HP:0030079": "#FFCCCC",  // light pink
+                                         "HP:0006775": "#FF0000",  // red 1
+                                         "HP:0001909": "#888888"   // gray
+                                       };
+            this._customCancerLabels = { "Breast": "Breast cancer",
+                                         "Ovarian": "Ovarian cancer",
+                                         "Colon": "Colorectal cancer",
+                                         "Uterus": "Uterus cancer",
+                                         "Prostate": "Prostate cancer",
+                                         "Pancreatic": "Pancreatic cancer",
+                                         "Melanoma": "Melanoma",
+                                         "Kidney": "Kidney cancer",
+                                         "Gastric": "Gastric cancer",
+                                         "Lung": "Lung cancer",
+                                         "Brain": "Brain cancer",
+                                         "Oesophagus": "Oesophagus cancer",
+                                         "Thyroid": "Thyroid cancer",
+                                         "Liver": "Liver cancer",
+                                         "Cervix": "Cervical cancer",
+                                         "Myeloma": "Myeloma",
+                                         "Leukemia": "Leukemia"
+                                       };
+            this._customLabelMapping = { "HP:0100013": "Breast",
+                                         "HP:0100615": "Ovarian",
+                                         "HP:0100273": "Colorectal",
+                                         "HP:0010784": "Uterus",
+                                         "HP:0100787": "Prostate",
+                                         "HP:0002894": "Pancreatic",
+                                         "HP:0012056": "Melanoma",
+                                         "HP:0009726": "Kidney",
+                                         "HP:0006753": "Gastric",
+                                         "HP:0100526": "Lung",
+                                         "HP:0030692": "Brain",
+                                         "HP:0100751": "Oesophagus",
+                                         "HP:0100031": "Thyroid",
+                                         "HP:0002896": "Liver",
+                                         "HP:0030079": "Cervical",
+                                         "HP:0006775": "Myeloma",
+                                         "HP:0001909": "Leukemia"
+                                       };
+
```

Comment:
I think it would be better to have just one mapping, something like
```
this._predefinedCancers = {
   "HP:0100013": { "shortName": "Breast", "longName": "Breast cancer", "defaultColor": "#e267a3" },
   ....
}
```
(this was true for the original implementation, but I guess even more relevant now - but clearly functionality is not affected

Reply:
Fixed.